README.md

```diff
@@ -1 +1,122 @@
-# Hello-Microverse
\ No newline at end of file
+<a name="readme-top">SET UP LINTERS AND CREATE PULL REQUEST</a>
+
+
+<div align="center">
+
+  <img src="images/images.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+
+</div>
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="https://zbahati.github.io/Hello-Microverse/">visit website</a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello Microverse]** is a web application that help us to set up the linters for the project that we will use to test the project.
+
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+#### this project have  one section that  is h1 with the message Hello Microverse.
+
+
+<!-- Features -->
+
+### Key Features <a name="https://zbahati.github.io/Hello-Microverse/">header-section features</a>
+
+
+- **[Test Linters of HTML and CSS ]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+> Add a link to your deployed project.
+
+- [Live Demo Link](https://zbahati.github.io/Hello-Microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+
+
+> Describe how a new developer could make use of your project.
+to get start with this testing for new developer we could first start with installing the following things linters,lighthouse, .hintrc,stylelintrc.json,
+
+### Prerequisites
+you need to have editor for writing  the codes and
+you need to have the gitHub, and Git to test how is the project
+
+### install the neccessary command to run the project
+install webhint using npm init -y, then install webhint by npm install --save-dev hint@7.x
+install lighthouse with npm install -g lighthouse
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Use the command for clone the report
+https://github.com/zbahati/Hello-Microverse.git
+
+<!--
+Example commands:
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+
+
+### Run tests
+
+You need to push the project to the github then the system will make the test
+First you need to create branch by using
+git checkout -b branch-name,
+then use the following command to push you branch
+
+- git add .
+- git commit -m 'add your message'
+- git push
+
+### Deployment
+
+You can deploy this project using GitHub by going on
+1.Setting
+2.pages
+3. publish the page
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Bahati Zirimwabagabo <a name="https://github.com/zbahati">Zbahati</a>
+
+
+👤 **Author**
+
+- GitHub: [@githubhandle](https://github.com/zbahati)
+- Twitter: [@twitterhandle](https://twitter.com/@b2_bahati)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/zirimwabagabo-bahati)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+
```

Comment:
- [ ] The previous reviewer requested some changes to the "future features", "contributing" and "support" sections, but I noticed that you deleted them completely, these sections are mandatory and should not be deleted please add them back and customise them properly.